Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -464,6 +468,29 @@ public class JavaScriptLifter: ComponentBase, Lifter {
                 
             case let op as Comment:
                 w.emitComment(op.content)
+
+            case is BeginCodeString:
+                if nestedCodeString > 0 {
+                    let count = Int(pow(2, Double(nestedCodeString)))-1
```

Comment:
Nice! Maybe make a small comment here where this series comes from, e.g. just something like "With every additional nesting level, ' becomes \' and every existing \ becomes \\, thus the power series".